### ISSUE TITLE
Loinc 57838-5:  "dRVVT W excess phospholipid (LA confirm)"

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#3630]
Suggest creating a new term for Loinc: 57838-5 [dRVVT W excess phospholipid (LA confirm)] 
New term label:
New term comment (if any):
Your biocurator ID for loinc2hpo (if desired): HPO:probinson

figure this out

**Comment by pnrobinson** [comment]
This is the Dilute Russell Viper Venom Time that is used for identifying lupus anticoagulants (LA) that do not prolong the activated partial thromboplastin time (APTT)

**Comment by pnrobinson** [comment]
@nicolevasilevsky @kingmanzhang 
THis is a test for https://hpo.jax.org/app/browse/term/HP:0025343
I will add a comment to that term.
